README.md

```diff
@@ -24,8 +24,65 @@ To cope with it, Zed's flatpak defaults can be changed to:
 When Zed's flatpak is running in the sandbox with no escape, it is not possible to execute commands on the host system.
 
 To execute commands on the host system, run inside the sandbox:
+
+```shell
+$ flatpak-spawn --host <COMMAND>
```

Comment:
Is the --host necessary with the JSON configuration file?

Reply:
Yes, otherwise flatpak-spawn will launch the command in the sandbox.